src/nominatim_api/localization.py

```diff
@@ -18,15 +17,15 @@ class Locales:
     """ Helper class for localization of names.
 
         It takes a list of language prefixes in their order of preferred
-        usage.
+        usage and comma separated name keys (Configuration.OUTPUT_NAMES).
     """
 
-    def __init__(self, langs: Optional[List[str]] = None):
-        self.config = Configuration(None)
+    def __init__(self, langs: Optional[List[str]] = None,
+                 names: Optional[str] = 'name:XX,name') -> None:
```

Comment:
Tiny nitpick: doesn't need to be of type 'Optional[str]' anymore, can now be 'str'. (same below)